Sources/SwiftDocC/DocumentationService/Models/Services/Convert/ConvertRequest.swift

```diff
@@ -288,4 +301,14 @@ extension ConvertRequest {
             self.character = character
         }
     }
+    
+    public struct ExpandedDocumentationRequirements: Codable {
+        public let accessControlLevels: [String]
+        public let canBeUnderscored: Bool
```

Comment:
It would be good to support checking SPI here as well but we can leave that to a future PR.